### PR TITLE
Andrew Jarrett: Complete Sprint

### PR DESCRIPTION
**Pull request by ahrjarrett**

### CODE REVIEW

**Comment by AlexCassell** [comment]
Good Job! The pop up is great! "Eat you some space food for great good! "  😂

File: Answers.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+1. How would you teach the basic ideas behind preprocessing to a friend?  Please don't copy and paste an answer here, use your own thoughts.
```
Comment (Xarfo) [inline Answers.md:1]: Great short answers, Andrew.

File: index.html [hunk 2]
```diff
@@ -15,16 +15,206 @@
 </head>
 
 <body>
+  <div class="nav">
+    <h1 class="headline" >
+      SpaceWalkers
+    </h1>
+    <div class="menu" >
```
Comment (Xarfo) [inline index.html:22]: Nicely nested anchored elements for the navigation.

File: less/layout.less [hunk 3]
```diff
@@ -5,4 +5,63 @@ html, body {
 h1, h2, h3, h4, h5 {
     font-family: 'Russo One', sans-serif;
     color: @secondary-color;
-}
\ No newline at end of file
+}
+
+a {
+    color: @secondary-color;
+    &:hover {
```
Comment (Xarfo) [inline less/layout.less:12]: Good use of variable and the nested hover effect.

File: less/index.less [hunk 4]
```diff
@@ -1 +1,4 @@
-// Make your imports here!
+@import 'bootstrap';
```
Comment (Xarfo) [inline less/index.less:1]: LESS files imported correctly and in order!

**Review by Xarfo — COMMENTED** [review]
Great job overall, Andrew!

Potential improvements:

- Make more descriptive commits.

**Pull request closed without merging**